.eslintrc.json

```diff
@@ -121,17 +132,4 @@
     "mocha": true,
     "es6": true
   }
-  // "globals": {
```

Comment:
keep these, they are used for jsdoc linting

Reply:
Ah, I was wondering what these were for, but why are they disabled?

Reply:
jsdoc linting is not enable for now

Reply:
Sounds like something we should fix :>